change/@fluentui-react-motion-b068209f-dd7d-43e0-a77a-f6ab0606659d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31602&commitId=ea80d1be1040c40e62a41c6379ccea38298bfb3a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>